org.moreunit.plugin/src/org/moreunit/elements/MissingClassTreeContentProvider.java

```diff
@@ -16,30 +19,41 @@
 
 public class MissingClassTreeContentProvider implements ITreeContentProvider
 {
+    private List<ICompilationUnit> units = new ArrayList<>();
```

Comment:
This is not a good Idea to store this kind of objects.It could lead to a memory leak. Can't you avoid those lists ?